bankaccount.go

```diff
@@ -38,17 +38,19 @@ type BankAccountListParams struct {
 
 // BankAccount represents a Stripe bank account.
 type BankAccount struct {
-	ID          string            `json:"id"`
-	Name        string            `json:"bank_name"`
-	Country     string            `json:"country"`
-	Currency    Currency          `json:"currency"`
-	Default     bool              `json:"default_for_currency"`
-	LastFour    string            `json:"last4"`
-	Fingerprint string            `json:"fingerprint"`
-	Status      BankAccountStatus `json:"status"`
-	Routing     string            `json:"routing_number"`
-	Deleted     bool              `json:"deleted"`
-	Customer    *Customer         `json:"customer"`
+	ID          			string            `json:"id"`
+	BankName    			string        		`json:"bank_name"`
+	Name        			string						`json:"name"`
+	AccountHolderType string						`json:"account_holder_type"`
```

Comment:
I think Remi got this on another commit already, but looks like there was some whitespace issues here.